build/source/dshare/ascii_util.f90

```diff
@@ -134,10 +135,10 @@ subroutine get_vlines(unt,vlines,err,message)
  ! do not know how many valid lines, so use linked lists
  implicit none
  ! declare dummy arguments
- integer(i4b),intent(in)              :: unt         ! file unit
- character(*),intent(out),allocatable :: vlines(:)   ! vector of character strings
- integer(i4b),intent(out)             :: err         ! error code
- character(*),intent(out)             :: message     ! error message
+ integer(i4b),intent(in)                          :: unt         ! file unit
+ character(len=linewidth),intent(out),allocatable :: vlines(:)   ! vector of character strings
```

Comment:
So - this is the heart of the matter, correct?

Reply:
Correct.